jsx/Form.js

```diff
@@ -195,6 +195,7 @@ class SearchableDropdown extends Component {
   render() {
     let required = this.props.required ? 'required' : null;
     let disabled = this.props.disabled ? 'disabled' : null;
+    let sortByValue = this.props.sortByValue;
```

Comment:
surely all these changes in Form.js should be accompanied by changes made to compiled js files?